src/state/transactions/updater.tsx

```diff
@@ -226,6 +249,33 @@ export default function Updater(): null {
               dispatch(checkedSubgraph({ chainId, hash }))
               break
             }
+            case 'Elastic Add liquidity': {
+              const res = await apolloClient.query({
+                query: PROMM_GET_POOL_VALUES_AFTER_MINTS_SUCCESS,
+                variables: {
+                  poolAddress: transaction.arbitrary.poolAddress.toLowerCase(),
+                },
+                fetchPolicy: 'network-only',
+              })
+              if (
+                !res.data?.pool?.mints &&
+                transaction.confirmedTime &&
+                new Date().getTime() - transaction.confirmedTime < 3600000
+              )
+                break
+              if (res.data.pool.mints.every((mint: { id: string }) => !mint.id.startsWith(transaction.hash))) break
+              const { totalValueLockedToken0, totalValueLockedToken1, totalValueLockedUSD, feeTier } = res.data.pool
+              mixpanelHandler(MIXPANEL_TYPE.ELASTIC_ADD_LIQUIDITY_COMPLETED, {
+                token_1_pool_qty: totalValueLockedToken0,
+                token_2_pool_qty: totalValueLockedToken1,
+                liquidity_USD: totalValueLockedUSD,
```

Comment:
do we track total value of pool or just value which user added?

Reply:
yes total value of the pool, to see how total value changed after liquid updated

Reply:
![image](https://user-images.githubusercontent.com/9592090/171783852-8d0866e3-d9fe-434b-bf19-5f4c86d41756.png)
in the definition of this field, confirmed with Zach, it is the pool's liquidity after the transaction not user liquidity. I already added tx hash for these events. Anyway, the current logic is FE will check subgraph until it sees the transaction in mints or burns list, until then it will fire the event, so I think it will not out of sync, and in case the transaction does not exist on subgraph somehow, it will force to fire after 1 hour without the field's value.